### PR TITLE
Animate donut and pie chart. 

### PR DESCRIPTION
**Pull request by gabycampagna**
Change layout for tooltip and charts. Add delay to tooltip (needs more work). New SCSS for styling tooltip.

### CODE REVIEW

**Comment by jrjohnson** [comment]
Deploy preview ready!

Built with commit e48200cd1e299b891e6e809b3bd7b2dc0b23b6b4

https://deploy-preview-2797--ilios-frontend.netlify.com

File: package.json [hunk 1]
```diff
@@ -21,7 +21,7 @@
     "ember-ajax": "2.5.6",
     "ember-can": "0.8.3",
     "ember-cli": "2.12.0",
-    "ember-cli-app-version": "^2.0.1",
+    "ember-cli-app-version": "^3.0.0",
```
Comment (jrjohnson) [inline package.json]: Changes to package.json should always be made in individual PRs so we can easily track down bugs if they arise later.  These updates are usually handled automatically by the greenkeeper bot.

File: package.json [hunk 2]
```diff
@@ -67,14 +67,14 @@
     "ember-data": "2.12.1",
     "ember-drag-drop": "0.4.2",
     "ember-exam": "0.6.0",
-    "ember-export-application-global": "^1.0.5",
+    "ember-export-application-global": "^2.0.0",
```
Comment (jrjohnson) [inline package.json]: Revert and let greenkeeper handle

File: package.json [hunk 3]
```diff
@@ -67,14 +67,14 @@
...
     "ember-font-awesome": "3.0.5",
     "ember-froala-editor": "2.4.2",
     "ember-i18n": "5.0.0",
     "ember-i18n-cp-validations": "3.0.1",
     "ember-inflector": "1.11.0",
     "ember-load": "0.0.11",
-    "ember-load-initializers": "^0.6.1",
+    "ember-load-initializers": "^1.0.0",
```
Comment (jrjohnson) [inline package.json]: Revert and let greenkeeper handle

File: package.json [hunk 4]
```diff
@@ -83,9 +83,9 @@
     "ember-one-way-controls": "2.0.0",
     "ember-pad": "1.2.2",
     "ember-papaparse": "0.0.3",
-    "ember-pikaday": "2.2.1",
+    "ember-pikaday": "2.2.2",
```
Comment (jrjohnson) [inline package.json]: Revert and let greenkeeper handle

File: package.json [hunk 5]
```diff
@@ -83,9 +83,9 @@
...
     "ember-promise-helpers": "1.0.2",
-    "ember-resolver": "^3.0.0",
+    "ember-resolver": "^4.0.0",
```
Comment (jrjohnson) [inline package.json]: Revert and let greenkeeper handle

File: package.json (hunk 1 above)
Comment (gabycampagna) [inline package.json]: Thanks, Jon. I am not even sure how it was changed. I committed my code, then I updated my fork, and the new JSON file was added to my local, then I had to to push it to my fork.  I am not sure where the JSON update came from, but sounds like it. I will close this PR and see if I can start over.

**Comment by jrjohnson** [comment]
Superseded by #2808

**Pull request closed without merging**